packages/babel-preset-gatsby/README.md

```diff
@@ -0,0 +1,43 @@
+# babel-preset-gatsby
+
+Gatsby uses the phenomenal project [Babel](https://babeljs.io/) to enable support for writing modern JavaScript — while still supporting older browsers. This package contains the default Babel setup for all Gatsby projects.
+
+## Packages
+
+- [`@babel/preset-env`](https://babeljs.io/docs/en/babel-preset-env)
+- [`@babel/preset-react`](https://babeljs.io/docs/en/babel-preset-react)
+- [`@babel/preset-flow`](https://babeljs.io/docs/en/babel-preset-flow)
+- [`@babel/plugin-proposal-class-properties`](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties)
+- [`@babel/plugin-proposal-optional-chaining`](https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining)
+- [`@babel/plugin-transform-runtime`](https://babeljs.io/docs/en/babel-plugin-transform-runtime#docsNav)
+
+## Usage
+
+Add a `.babelrc` file with the following content to the root of your project:
+
+```json
+{
+  "presets": ["gatsby"]
```

Comment:
I'll comment one more time that I'd love these to be `babel-preset-gatsby`, but you get the idea :)